react/new/src/components/ThemeControls.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Select,
+  Switch,
+  Stack,
+  useMantineColorScheme,
+  Group,
+  Box,
+} from '@mantine/core'
+import { useTheme } from '../contexts/ThemeContext'
+import { NHLTeam } from '../types/theme'
+import { NHL_TEAM_COLORS } from '../constants/nhlColors'
+import { NHL_TEAM_NAMES } from '../constants/nhlTeams'
+
+export function ThemeControls() {
+  const { theme, setThemeMode, setTeamTheme } = useTheme()
+  const { setColorScheme } = useMantineColorScheme()
+
+  const handleThemeToggle = (checked: boolean) => {
+    const newMode = checked ? 'dark' : 'light'
```

Comment:
Remind me, will these persist via local cache, or is that something we'll need accounts for in order to save preferences?